<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Scheduler</class>
 <widget class="QDialog" name="Scheduler">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scheduler</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QListWidget" name="scheduled">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>421</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="remove">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>50</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLabel" name="painterLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>91</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Command:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="parameters">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>170</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="commands">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="painterLabel_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Parameters (separated by space):</string>
   </property>
  </widget>
  <widget class="QPushButton" name="run">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>230</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Run!</string>
   </property>
  </widget>
  <widget class="QLabel" name="help">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>451</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="up">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>^</string>
   </property>
  </widget>
  <widget class="QPushButton" name="down">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>90</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>v</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>310</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>150</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>311</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>For $i in:</string>
   </property>
   <widget class="QListWidget" name="loopValues">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="loopRemove">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="loopValue">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="loopAdd">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="log">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>461</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>scheduled</tabstop>
  <tabstop>up</tabstop>
  <tabstop>remove</tabstop>
  <tabstop>down</tabstop>
  <tabstop>commands</tabstop>
  <tabstop>parameters</tabstop>
  <tabstop>add</tabstop>
  <tabstop>loopValues</tabstop>
  <tabstop>loopRemove</tabstop>
  <tabstop>loopValue</tabstop>
  <tabstop>loopAdd</tabstop>
  <tabstop>run</tabstop>
  <tabstop>stop</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
